modules/apps/asset/asset-display-page-item-selector-web/src/main/java/com/liferay/asset/display/page/item/selector/web/internal/servlet/taglib/clay/LayoutPageTemplateEntryVerticalCard.java

```diff
@@ -41,6 +41,12 @@ public LayoutPageTemplateEntryVerticalCard(
 		_httpServletRequest = PortalUtil.getHttpServletRequest(renderRequest);
 	}
 
+	@Override
+	public String getCssClass() {
+		return "card-interactive card-interactive-secondary " +
```

Comment:
this makes me think... Is this a case of "interactive" a.k.a navigation card?

Reply:
very good question @carloslancha: it used to be done with a `<clay:vertical-card-v2 />` but since it has `interactive` in it's CSS classes this could the case.

This is how it looks like (in `master`)

![cards](https://user-images.githubusercontent.com/5572/99267665-888fef00-2824-11eb-9dd1-ac7fc9b2e635.png)

It doesn't seem to behave like an navigation card [defined here](https://liferay.design/lexicon/core-components/cards/cards-navigation/) though, so I'd say it's OK to replace it with a the new `<clay:vertical-card />`

Reply:
That looks like a navigation card attempt, they probably did it before navigation card was created and just improvised a workaround. I'd leave it as it is (using `vertical-card`) but asking Echo if the pattern they're trying to follow is covered by a navigation card, and if so, encourage them to update de usage once we create the navigation card tag, since this pattern is neither one or the other.

Reply:
Sounds like a good idea, I'm OK with that.